### PR TITLE
Base APP UI

### PR DESCRIPTION
**Pull request by AliMD**
## Description

Base UI base on ioinic framework

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Open an issue for bug report or feature request and mention it to this PR.
- [x] The commit message follows our guidelines: <https://github.com/AliMD/alwatr/blob/main/CONTRIBUTING.md#commit-message-format>
- [x] All methods have proper description in `jsdocs` format with `@example` section.
- [x] Docs have been added / updated (for bug fixes / features).
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: src/elements/page-home.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+          <ion-title>عنوان برنامه ...</ion-title>
+        </ion-toolbar>
+      </ion-header>
+
+      <ion-content class="padding">
+        <div class="menu-container">
+          <a
+            class="cart-image"
+            href=${router.makeUrl({sectionList: ['aaa']})}
+            style="background-image: url('/images/1005.jpeg'); height: 30vw;">
```
Comment (MM25Zamanian) [inline src/elements/page-home.ts:119]: You can use ion-thumbnail ؟

File: src/elements/page-home.ts [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  override render(): TemplateResult {
+    return html`
+      <ion-header>
+        <ion-toolbar>
+          <ion-title>عنوان برنامه ...</ion-title>
+        </ion-toolbar>
+      </ion-header>
+
+      <ion-content class="padding">
+        <div class="menu-container">
```
Comment (MM25Zamanian) [inline src/elements/page-home.ts:115]: You can use ion-grid?

Comment (AliMD) [reply src/elements/page-home.ts:115]: no!

File: src/elements/page-home.ts (hunk 1 above)
Comment (AliMD) [reply src/elements/page-home.ts:119]: Why?!

**Pull request merged**